src/strerrno.cpp

```diff
@@ -30,7 +30,7 @@ pj_err_list[] = {
     "acos/asin: |arg| >1.+1e-14",                                      /* -19 */
     "tolerance condition error",                                       /* -20 */
     "conic lat_1 = -lat_2",                                            /* -21 */
-    "lat_1 >= 90",                                                     /* -22 */
+    "lat >= 90",                                                       /* -22 */
```

Comment:
Maybe this would be better as "lat_1 or lat_2 >= 90"? "lat" can be mistaken for the input latitude.

Reply:
I wanted this to be generic, and potentially be used for lat_0 as well. So 'lat_0, lat_1 or lat_2 >= 90' ?

Reply:
agreed